### PR TITLE
Micro batch followup

### PR DESCRIPTION
**Pull request by majolo**
Simplifying and some small adjustments after the last PR was automerged.

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cliveseldon**
You can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: python/seldon_core/batch_processor.py [hunk 1]
```diff
@@ -309,18 +308,10 @@ def _send_batch_predict_multi_request(
...
-        data_np = np.array(data)
-        concat = data_np
-        for i, raw_data in enumerate(input_data):
-            # Already added first item.
-            if i == 0:
-                continue
-            data = json.loads(raw_data[2])
-            data_np = np.array(data)
-            concat = np.concatenate((concat, data_np))
+        loaded = [json.loads(raw_data[2]) for raw_data in input_data]
```
Comment (RafalSkolasinski) [inline python/seldon_core/batch_processor.py:311]: Nice one, looks much better!

**Review by RafalSkolasinski — APPROVED** [review]
Nice!

/test integration

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[RafalSkolasinski](https://github.com/SeldonIO/seldon-core/pull/3306#pullrequestreview-686458920 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [RafalSkolasinski]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by axsaucedo** [comment]
Another  thing I would add is perhaps expanding on the documentation for mini-batching (thre is a batch processing section in the docs)

**Comment by majolo** [comment]
@axsaucedo I did that in the last PR, added docs about it 👍🏻

**Pull request merged**